application-analytics-ui/src/main/resources/Analytics/DefaultDashbord.xml

```diff
@@ -0,0 +1,147 @@
+      <defaultViewSheet/>
+      <defaultEditSheet/>
+      <defaultWeb/>
+      <nameField/>
+      <validationScript/>
+      <ListOfWidgets>
+        <cache>0</cache>
+        <customDisplay/>
+        <defaultValue/>
+        <disabled>0</disabled>
+        <displayType>radio</displayType>
+        <freeText/>
+        <hint/>
+        <largeStorage>0</largeStorage>
+        <multiSelect>1</multiSelect>
+        <name>ListOfWidgets</name>
+        <number>4</number>
+        <picker>1</picker>
+        <prettyName>listOfWidgets</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator> </separator>
+        <separators> ,|</separators>
+        <size>1</size>
+        <sort/>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <values>Random | Words | For | Now</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </ListOfWidgets>
+      <description>
+        <contenttype>---</contenttype>
+        <customDisplay/>
+        <disabled>0</disabled>
+        <editor>---</editor>
+        <hint/>
+        <name>description</name>
+        <number>3</number>
+        <picker>1</picker>
+        <prettyName>Description</prettyName>
+        <rows>5</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </description>
+      <end>
+        <customDisplay/>
+        <dateFormat>dd/MM/yyyy</dateFormat>
+        <disabled>0</disabled>
+        <emptyIsToday>0</emptyIsToday>
+        <hint/>
+        <name>end</name>
+        <number>2</number>
+        <picker>1</picker>
+        <prettyName>End Date</prettyName>
+        <size>20</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.DateClass</classType>
+      </end>
+      <start>
+        <customDisplay/>
+        <dateFormat>dd/MM/yyyy</dateFormat>
+        <disabled>0</disabled>
+        <emptyIsToday>0</emptyIsToday>
+        <hint/>
+        <name>start</name>
+        <number>1</number>
+        <picker>1</picker>
+        <prettyName>Start Date</prettyName>
+        <size>20</size>
+        <unmodifiable>0</unmodifiable>
+        <validationMessage/>
+        <validationRegExp/>
+        <classType>com.xpn.xwiki.objects.classes.DateClass</classType>
+      </start>
+    </class>
+    <property>
+      <ListOfWidgets>
+        <value>Random </value>
+        <value> Words </value>
+        <value> For </value>
+        <value> Now</value>
+      </ListOfWidgets>
+    </property>
+    <property>
+      <description>This is a default dashbord.</description>
+    </property>
+    <property>
+      <end>2023-07-17 00:00:00.0</end>
+    </property>
+    <property>
+      <start>2023-07-17 00:00:00.0</start>
```

Comment:
I think you could leave these empty,  which will be interpreted from your code after as no specific timeframe (I'm not sure if Matomo expects something specific as query parameters when you don't what to apply a timeframe filter)